tools/build_defs/shell_toolchain/toolchains/impl/default_commands.bzl

```diff
@@ -73,10 +73,12 @@ if [[ -f $1 ]]; then
   return 0
 fi
 
-local children=$(find $1 -maxdepth 1 -mindepth 1)
-for child in $children; do
-  ##symlink_to_dir## $child $target
-done
+if [[ -d $1 || -L $1 ]]; then
```

Comment:
`[[` is a `bash` extension; as there are already `bash`-only constructs in the `text`, I assume it is OK, but I couldn't find the documentation about whether this is called by a generic shell (like `sh`) or by `bash` specifically (the directory name `shell_toolchain` seems to hint at the former).

Reply:
Thank you for the remark; bash is assumed, will clarify that.